### PR TITLE
add EOL for 1.26

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This adds the EOL date and also updates our scripting for yq4 (hooray, we arrived in 2021!)

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by xrstf** [comment]
/hold

@xmudrii says the date is wrong.

**Review by xmudrii — APPROVED** [review]

**Comment by xrstf** [comment]
/hold cancel

just to unblock the periodic job

**Pull request merged**
